backend/src/main/resources/reference.conf

```diff
@@ -3,14 +3,16 @@ bootzooka {
 
   db {
     h2 {
+      queueSize = -1
       dataSourceClass = "org.h2.jdbcx.JdbcDataSource"
       properties = {
         url = "jdbc:h2:file:./data/bootzooka"
       }
     }
     postgres {
+      queueSize = -1
       dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
-      maxConnections = 16
+      maxConnections = 20
```

Comment:
@adamw I can instead leave the number of connections at 16 and change the number of threads to 16